src/routes/auth/magic-link.ts

```diff
@@ -62,27 +61,9 @@ async function magicLink({ query }: Request, res: Response): Promise<unknown> {
     throw Boom.unauthorized('Invalid or expired token.')
   }
 
-  const jwt_token = createHasuraJwt(account)
-  const jwt_expires_in = newJwtExpiry
-  const user: UserData = {
-    id: account.user.id,
-    display_name: account.user.display_name,
-    email: account.email,
-    avatar_url: account.user.avatar_url
-  }
-  const session: Session = { jwt_token, jwt_expires_in, user }
-  if (!useCookie) session.refresh_token = refresh_token
-
-  if (action === 'log-in') {
-    return res.redirect(`${APPLICATION.REDIRECT_URL_SUCCESS}?refresh_token=${refresh_token}`)
-  } else if (action === 'sign-up') {
-    if(APPLICATION.REDIRECT_URL_SUCCESS) {
-      return res.redirect(APPLICATION.REDIRECT_URL_SUCCESS.replace('JWT_TOKEN', token))
```

Comment:
@komninoschat Was `.replace('JWT_TOKEN', token)` for anything special? I don't understand why we had it here.